src/components/input/index.tsx

```diff
@@ -99,13 +104,39 @@ const Input = forwardRef<HTMLInputElement, Props>(
       visible: !!value,
       disabled,
       onClear: () => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        ;(inputRef.current as any).value = ""
```

Comment:
is there a more concrete type than any that we can use? some lines down, we used `HTMLInputElement`?